### DISCUSSION TITLE
How does Prisma sanitize Json data?

### DISCUSSION THREAD

**Discussion by rjtngit** [prisma/prisma#7635] [Q&A]
Are there any vulnerabilities with inserting JSON data generated by the user's client directly into the database? Do I need to sanitize it? How does Prisma handle this? 

I can't find any documentation about this. If someone could point me to the docs that would be great!

**Comment by ryands17** [comment]
@rempelj 👋 

What specific concerns do you have? All Prisma methods sanitise data to prevent stuff like SQL injection so no configuration is needed from your end.

You can read about this [here](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#sql-injection).

**Reply by rjtngit** [reply]
This is what I'm looking for. thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
